src/components/pages/ShopDetail.tsx

```diff
@@ -77,15 +111,17 @@ function ShopDetail() {
     rating: number;
     place_id: string;
   };
+
   // Railsからparams id と同じidのデータを取得
   useEffect(() => {
     // isFirstRef.current = false;
-    axios.get(`${process.env.REACT_APP_BACK_ORIGIN}/api/v1/shops/${id}`)
-         .then(res => {setShop(res.data)
-               console.log("Rails Api からデータを取得");
-               console.log(res.data);
-              })
-         .catch(error => console.log(error))
+    axios.get(`${process.env.REACT_APP_BACK_ORIGIN_DEVELOPMENT}/api/v1/shops/${id}`)
+      .then(res => {
+        setShop(res.data.shop)
+        setFavoriteData(res.data.favorite)
```

Comment:
favorite が undefined になる可能性があるので以下の方が良いかもですね！
```suggestion
        !!res.data.favorite && setFavoriteData(res.data.favorite)
```